build.gradle

```diff
@@ -1,15 +1,46 @@
-allprojects {
-    apply plugin: 'java'
-    apply plugin: 'eclipse'
+apply plugin: 'application'
+apply plugin: 'com.esri.arcgisruntime.java'
 
+buildscript {
     repositories {
-        mavenCentral()
-        maven {
-            url 'http://android:8080/artifactory/arcgis'
-        }
+        /*maven {
+            url 'https://esri.bintray.com/arcgis'
+        }*/
+        mavenLocal()
     }
-
     dependencies {
-        compile 'com.esri.arcgisruntime:arcgis-java:100.0.0-1450'
+        classpath 'com.esri.arcgisruntime:gradle-arcgis-java-plugin:1.0.0-SNAPSHOT'
```

Comment:
Where are you pulling this from?  Dependency cannot be retrieved so import to IntelliJ fails.